src/common.jl

```diff
@@ -124,10 +124,150 @@ function calculate_BA_matrix(
     return BA
 end
 
+function calculate_BA_matrix_full(
+    branches,
+    bus_lookup::Dict{Int, Int})
+    BA_I = Int[]
+    BA_J = Int[]
+    BA_V = Float64[]
+
+    for (ix, b) in enumerate(branches)
+        if isa(b, PSY.DCBranch)
```

Comment:
@alefcastelli delete this check. By calling the PNM function `get_ac_branches` when passing the arguments to the function is should be addressed.

Reply:
that is part of a function that has been removed in the latest commit